src/codeu/chat/server/Server.java

```diff
@@ -206,7 +209,10 @@ public void run() {
           final int type = Serializers.INTEGER.read(connection.in());
           final Command command = commands.get(type);
 
-          if (command == null) {
+          if (type == NetworkCode.SERVER_INFO_REQUEST) {
+            Serializers.INTEGER.write(connection.out(), NetworkCode.SERVER_INFO_RESPONSE);
+            Uuid.SERIALIZER.write(connection.out(), info.version);
```

Comment:
This code is not meant to define new commands. This should be in the constructor above as a new `commands.put` statement, something like:

`this.commands.put(NetworkCode.SERVER_INFO_REQUEST, new Command() {`